packages/demand-bloc-payment/src/providers/BraintreeProvider.ts

```diff
@@ -132,6 +150,14 @@ export class BraintreeProvider implements Provider {
     }
   }
 
+  private hasCardBinAllowlist(): boolean {
+    return Array.isArray(this.options.allowedBinValues) && this.options.allowedBinValues.length > 0
+  }
+
+  private isCardBinAllowed(bin: string | undefined): boolean {
+    return this.hasCardBinAllowlist() ? !!bin && !!this.options.allowedBinValues?.includes(bin) : true
```

Comment:
`!!` is redundant here `!!this.options.allowedBinValues?.includes(bin)`

Reply:
TS issue without `!!` - expects boolean, not `boolean | undefined`